input/kafkamdm/kafkamdm.go

```diff
@@ -197,16 +212,16 @@ func (k *KafkaMdm) Start(metrics mdata.Metrics, metricIndex idx.MetricIndex, usg
 }
 
 // this will continually consume from the topic until k.stopConsuming is triggered.
-func (k *KafkaMdm) consumePartition(topic string, partition int32, partitionOffset int64) {
+func (k *KafkaMdm) consumePartition(topic string, partition int32, initialOffset int64) {
 	k.wg.Add(1)
 	defer k.wg.Done()
 
-	pc, err := k.consumer.ConsumePartition(topic, partition, partitionOffset)
+	pc, err := k.consumer.ConsumePartition(topic, partition, initialOffset)
 	if err != nil {
 		log.Fatal(4, "kafka-mdm: failed to start partitionConsumer for %s:%d. %s", topic, partition, err)
 	}
-	log.Info("kafka-mdm: consuming from %s:%d from offset %d", topic, partition, partitionOffset)
-	currentOffset := partitionOffset
+	log.Info("kafka-mdm: consuming from %s:%d from offset %d", topic, partition, initialOffset)
+	currentOffset := initialOffset
```

Comment:
yes, we can just get rid of initialOffset and refer to it as "currentOffset" throughout the function.